test3.js

```diff
@@ -0,0 +1,191 @@
+/**
+ * Model file to handle all repository actions
+ */
+const { repoSchema, tempRepoSchema } = require('../schema/index');
+const { Logger, DB } = require('../helpers/index');
+const path = require('path');
```

Comment:
Codacy has a fix for the issue: [Replace `'path'` with `"path"`](https://app.codacy.com/gh/thamizhMsys/testRepo6/pullRequest?prid=9650715)
```suggestion
const path = require("path");
```